crates/syntax/src/ast/make.rs

```diff
@@ -14,6 +14,8 @@ use stdx::{format_to, never};
 
 use crate::{ast, utils::is_raw_identifier, AstNode, SourceFile, SyntaxKind, SyntaxToken};
 
+use super::WhereClause;
```

Comment:
nit, we prefer using `crate::` import paths instead of relative `super` ones. if this is from r-a's autoimport feature you mightve misconfigured something (it should usually try to match the import schemes of a file unless explicitly disabled)

Reply:
on a second note, we usually don't import ast nodes directly and instead refer to them via the `ast::` prefix as is done elsewhere in this module